Cargo.toml

```diff
@@ -60,6 +60,7 @@ pam = ["pam-auth"]
 oauth2 = ["yup-oauth2", "hyper-rustls"]
 cloud_storage = ["oauth2", "mime", "url", "hyper", "serde", "serde_json"]
 rest = ["hyper", "url", "http", "serde", "serde_json"]
```

Comment:
thinking out loud. if we rename jsonfile to say jsonfileauth we should probably do the same for the rest feature.

Reply:
I agree, i'll be including this change as well.

Reply:
will make a separate pull request for this